tests/analyzer_tech_debt.txt

```diff
@@ -127,4 +127,5 @@
 02790_optimize_skip_unused_shards_join
 01940_custom_tld_sharding_key
 02815_range_dict_no_direct_join
+00826_cross_to_inner_join_with_reordering
```

Comment:
What is different comparing to new analyzer? Is it only column names? Can we check only values and have the same behaviour?